packages/react-integration/demo-app-ts/src/components/demos/ContextSelectorDemo/ContextSelectorDemo.tsx

```diff
@@ -10,12 +10,12 @@ interface ContextSelectorState {
 
 export class ContextSelectorDemo extends React.Component<{}, ContextSelectorState> {
   items: string[];
-  onToggle: (event: any, isOpen: any) => void;
+  onToggle: (event: any, isOpen: boolean) => void;
```

Comment:
Instead of typing the event param as `any`, can we omit the type altogether for the time being? This will make it easier to identify places where we intentionally want to use the `any` type vs places where the types just haven't been defined yet.

Reply:
Removed event type